cloudevents/extensions/data-classification.md

```diff
@@ -0,0 +1,89 @@
+# Data Classification Extension
+
+CloudEvents might contain payload which is subjected to data protection
+regulations like GDPR or HIPAA. For intermediaries and consumers knowing how
+event payload is classified, which data protection regulation applies and how
+payload is categorized, enables compliant processing of an event.
+
+This extension defines attributes to describe to
+[consumers](../spec.md#consumer) or [intermediaries](../spec.md#intermediary)
+how an event and its payload is classified, category of the payload and any
+applicable data protection regulations.
+
+These attributes are intended for classification at an event and payload level
+and not at a `data` field level. Classification at a field level is best defined
+in the schema specified via the `dataschema` attribute.
+
+## Notational Conventions
+
+As with the main [CloudEvents specification](../spec.md), the key words "MUST",
+"MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+However, the scope of these key words is limited to when this extension is used.
+For example, an attribute being marked as "REQUIRED" does not mean it needs to
+be in all CloudEvents, rather it needs to be included only when this extension
+is being used.
+
+## Attributes
+
+### dataclassification
+
+- Type: `String`
+- Description: Data classification level for the event payload within the
+  context of a `dataregulation`. In situations where `dataregulation` is
+  undefined or the data protection regulation does not define any labels, then
+  recommended labels are: `public`, `internal`, `confidential`, or
+  `restricted`.
+- Constraints:
+  - REQUIRED
+
+### dataregulation
+
+- Type: `String`
+- Description: A comma-delimited list of applicable data protection regulations.
+  For example: `GDPR`, `HIPAA`, `PCI-DSS`, `ISO-27001`, `NIST-800-53`, `CCPA`.
+- Constraints:
+  - OPTIONAL
+  - if present, MUST be a non-empty string
```

Comment:
anal but... since it's a string with commas that need to be parsed, we may want to add something like "leading and trailing spaces around each entry MUST be ignored. Spaces within an entry MAY exist but MUST be reduced down to a single space for comparison purposes".  Are spaces allowed in the entries?

Reply:
Correct spaces are not allowed